src/util.ts

```diff
@@ -753,3 +753,14 @@ export async function prepareXchainFromAmountCalculation({
 		return null
 	}
 }
+
+export function normalizePath(url: string): string {
```

Comment:
TODO: can we comment functions like  these with a brief objective description? e.g. `// removes duplicate forward slashes`

Reply:
makes parsing code much easier

Reply:
I agree